package.nls.json

```
@@ -55,7 +55,7 @@
     "AWS.stepFunctions.publishWizard.stateMachineNameToUpdate.noStateMachines.detail": "Create a state machine before proceeding. See documentation for details.",
     "AWS.stepFunctions.publishWizard.stateMachineNameToUpdate.title": "Select state machine to update ({0})",
     "AWS.stepFunctions.render": "Render graph",
-    "AWS.command.samVisualize.renderTemplate": "Render CloudFormation template",
+    "AWS.command.samVisualize.renderTemplate": "Render CloudFormation Template graph",
```

Looking at existing capitalization in VS Code, should likely be:

```suggestion
    "AWS.command.samVisualize.renderTemplate": "Render CloudFormation Template Graph",
```

---

[The state machine render function doesn't seem to have it](https://github.com/aws/aws-toolkit-vscode/blob/e37c484103335ec8584c3e0788e5c3c8d5d2e5f3/package.nls.json#L194)

But if others do, I personally think capitalized `Graph` looks better